src/javascript/app_2/Utils/URL/__tests__/base_name.spec.js

```diff
@@ -0,0 +1,16 @@
+import { expect }   from 'chai';
+import getBaseName  from '../base_name';
+
+describe('base_name.js', () => {
+    describe('.getBaseName', () => {
+        it('should return basename from url', () => {
+            global.window.location.pathname = 'app/en/statement';
+            expect(getBaseName()).to.eql('/app/en/');
```

Comment:
@ahkuma better to use something else rather than the default base value (to make sure that it goes through this statement `if (basename && basename.length) {
        return basename[1];
    }`)